lib/qless/worker/base.rb

```diff
@@ -15,13 +15,16 @@ module Workers
 
     class BaseWorker
       attr_accessor :output, :reserver, :log_level, :interval, :paused,
-                    :options
+                    :options, :sighup_handler
 
       def initialize(reserver, options = {})
         # Our job reserver and options
         @reserver = reserver
         @options = options
 
+        # SIGHUP handler
+        @sighup_handler = options.fetch(:sighup_handler) { lambda { } }
```

Comment:
`log_stack_trace` didn't work.  That's why we are changing.  And this as brand new as of last week.